packages/cli/lib/services/dryrun.service.ts

```diff
@@ -243,10 +258,8 @@ export class DryRunService {
             }
 
             if (options.saveResponses) {
-                const responseDirectoryPrefix = process.env['NANGO_MOCKS_RESPONSE_DIRECTORY'] ?? '';
-                const directoryName = `${responseDirectoryPrefix}${providerConfigKey}`;
-                responseSaver.ensureDirectoryExists(`${directoryName}/mocks/${syncName}`);
-                const filePath = `${directoryName}/mocks/${syncName}/input.json`;
+                responseSaver.ensureDirectoryExists(saveResponsesSyncDir);
+                const filePath = `${saveResponsesSyncDir}/input.json`;
```

Comment:
@khaliqgant where are those `responseSaver` files being used. Do I need to modify something else? Thank you